iDiff/README.md

```diff
@@ -0,0 +1,167 @@
+# iDiff
+
+[![Build
+Status](https://travis-ci.org/GoogleCloudPlatform/runtimes-common.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/runtimes-common)
+[![codecov](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common)
+
+## What is iDiff?
+
+iDiff is an image differ command line tool.  iDiff can diff two images along several different criteria, currently including:
+- Docker Image History
+- Image file system
+- apt-get installed packages
+- pip installed packages
+- npm installed packages
+
+This tool can help you as a developer better understand what is changing within your images and better understand what your images contain.
+
+## Installation
+
+TODO: add how to install
+
+## Quickstart
+
+To use iDiff you need two Docker images (in the form of an ID, tarball, or URL from a repo).  Once you have those images you can run any of the following differs:
+
+```
+iDiff <img1> <img2>     [Run all differs]
+iDiff <img1> <img2> -d  [History]
+iDiff <img1> <img2> -f  [File System]
+iDiff <img1> <img2> -p  [Pip]
+iDiff <img1> <img2> -a  [Apt]
+iDiff <img1> <img2> -n  [Node]
+```
+
```

Comment:
Default all differs is the first line of examples.  Good point on multiple differs.